src/main/java/org/electrumj/dto/transactionget/TxInput.java

```diff
@@ -14,4 +16,6 @@
     private String txId;
     @JsonProperty("vout")
     private int index;
+    @JsonProperty("txinwitness")
+    private List<String> txinwitness;
```

Comment:
Could you make the field name camel case and add a test that gets a segwit tx so we can see how this works?